bin/wp

```diff
@@ -12,8 +12,4 @@ CLI="wp $COMMAND"
 # Mimic core `wp-cli`'s behavior of using `less` for `help` output.
 PAGER=$(if [ -z "$COMMAND" ] || [[ "$COMMAND" == \'help* ]]; then echo "less -R"; else echo "cat"; fi)
 
-if [ -n "$STUDIO_APP_DIRECTORY" ] && [ -n "$STUDIO_EXE_PATH" ]; then
-  "$STUDIO_EXE_PATH" "$STUDIO_APP_DIRECTORY" --cli="$CLI" | $PAGER
```

Comment:
This is no longer needed as the `STUDIO_APP_PATH` will contain the right path:
- Development builds (generated by `npm start`): `<PROJECT_PATH>/node_modules/electron/dist/Electron.exe <PROJECT_PATH>`
- Packaged builds (generated by `npm run package`): `<PROJECT_PATH>/out/**/Studio.exe`